### PR TITLE
Fix outline panel blocking clicks incorrectly

### PR DESCRIPTION
**Pull request by vezwork**
Co-authored-by: Fülöp Kovács <kovacs.fulop@gmail.com>

|  Before |  After |
|---|---|
|  image |  image |

### CODE REVIEW

File: theatre/studio/src/panels/OutlinePanel/BaseItem.tsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ const Header = styled(BaseHeader)`
   // Hit zone
   &:before {
     position: absolute;
-    inset: -1px -20px;
+    inset: -1px 0 -1px 0;
```
Comment (colelawrence) [inline theatre/studio/src/panels/OutlinePanel/BaseItem.tsx:68]:
```suggestion
    inset: -1px 0;
```

Comment (colelawrence) [reply theatre/studio/src/panels/OutlinePanel/BaseItem.tsx:68]: Also, does this need any kind of comment explaining why it wasn't working before? I suppose not, but maybe @AndrewPrifer knows if he impl'd

**Review by colelawrence — CHANGES_REQUESTED** [review]
nit

File: theatre/studio/src/panels/OutlinePanel/BaseItem.tsx (hunk 1 above)
Comment (vezwork) [reply theatre/studio/src/panels/OutlinePanel/BaseItem.tsx]: I'm not sure why this had `-20px` before. Removing it (making it `inset: -1px 0;`) causes the clickable area of the outline items to match their visual size.

**Review by colelawrence — APPROVED** [review]

File: theatre/studio/src/panels/OutlinePanel/OutlinePanel.tsx [hunk 2]
```diff
@@ -15,7 +14,7 @@ const Container = styled.div<{pin: boolean}>`
   position: absolute;
   left: 8px;
   z-index: ${panelZIndexes.outlinePanel};
-  ${pointerEventsAutoInNormalMode};
```
Comment (AriaMinaei) [inline theatre/studio/src/panels/OutlinePanel/OutlinePanel.tsx:18]: Outcome of removing `pointerEventsAutoInNormalMode`:

Screenflick Movie 115

**Review by AriaMinaei — CHANGES_REQUESTED** [review]

File: theatre/studio/src/panels/OutlinePanel/OutlinePanel.tsx (hunk 2 above)
Comment (vezwork) [reply theatre/studio/src/panels/OutlinePanel/OutlinePanel.tsx:18]: @AriaMinaei I tested, and it seems the behaviour you are showing (hover styling while dragging) was already present on `main` in the outline items before this PR's changes.

Comment (vezwork) [reply theatre/studio/src/panels/OutlinePanel/OutlinePanel.tsx:18]: If you intended hover styles to be disabled while dragging, I can add a fix for that to this PR.

Comment (AriaMinaei) [reply theatre/studio/src/panels/OutlinePanel/OutlinePanel.tsx:18]: @vezwork oh yeah please do

Comment (vezwork) [reply theatre/studio/src/panels/OutlinePanel/OutlinePanel.tsx:18]: Fixed the hover styles and another issue where useCssCursorLock was not being used when dragging part of the sequence panel @AriaMinaei !

**Review by AriaMinaei — APPROVED** [review]

**Pull request merged**